source/guides/playbooks.rst

```diff
@@ -12,24 +12,24 @@ Before You Begin
    /playbooks/overview.rst
    /playbooks/getting-started.rst
    
-Planning
---------
+Work with Playbooks
```

Comment:
Work or Working?

Reply:
Work, to match Messaging although I am open to "Working" as well if updating Messaging is an option?

Reply:
Pre v6.0 is a great time for you and I to align on the use of gerunds going forward in titles for all new content development. 

Do we want to define gerund use per title type (H1 through H4)? Or do we want to use one versus the other consistently for all title types? I'm open to either option? I'm strongly leaning towards simplicity and applying either gerunds or no gerunds across the doc set going forward. Thoughts?

Reply:
I'm 5/5 on consistency. We've always leaned toward gerunds and I think it'd make life easier if we used them throughout as opposed to one standard for H1 and a different standard for H2 onwards.

It would be less work to change these to gerunds than the opposite from what I can see from a brief glance through the docs. That said, not all our titles lend themselves to gerunds so I guess the rule should be "when a title has an action, use a gerund" so that pages like "Release notes" won't have to be changed to "Releasing notes". ;)